### DISCUSSION TITLE
人生第一个双金！！！

### DISCUSSION THREAD

**Discussion by 0AIER0** [share121/inter-knot#1470] [其他]
![QQ20240728-202812](https://github.com/user-attachments/assets/ac3a75f3-bda7-4cce-b394-99b355829e06)
如图，玩了这么久米游，第一次双金！

**Comment by hzjwork** [comment]
你不许玩绝区零😡

**Comment by Jiucheng258** [comment]
P的，一眼假不接受反驳。😭

**Comment by ximulili** [comment]
恭喜！

**Comment by capacitor1** [comment]
😡不信，除非把号给我看看

**Comment by dominic121-dot** [comment]
一眼假
